ipc/cli/src/commands/subnet/create.rs

```diff
@@ -96,4 +97,9 @@ pub struct CreateSubnetArgs {
         help = "Minimum fee for cross-net messages in subnet (in whole FIL)"
     )]
     pub min_cross_msg_fee: f64,
+    #[arg(
+        long,
```

Comment:
the conversion to `long` will be done automatically by the CLI library we use, right?

Reply:
This `long` what specifies is that the cli will only expect the long form of the flag name i.e. `--permissioned` not the short version which would be `-p`.